src/Components/HomePage/HomePage.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import './HomePage.css'
 
 const HomePage = ({filepath, getRandomDog}) => {
-  let rename;
   const randomPathBreed = filepath.split('/')[4]
```

Comment:
Mostly just curious as to what's going on here. Are these files within your repository? If so it feels like there might be a more readable approach to accessing those? And if the random dog is being generated on the home page, then maybe(?) it'd be easier to follow to just manage that state within the homePage component rather than (presumably) within App. I feel like that might help limit the number of props being passed back and forth and make each page a little more modular and easier to refactor as a separable/distinct set of code.